controller/service.go

```diff
@@ -39,55 +55,79 @@ func (bs *BirthdayStruct) asBirthdayObject() *pb.BirthdayObject {
 	return birthdayO
 }
 
+// CreateBirthday is creating a birthday object
 func (s *Service) CreateBirthday(ctx context.Context, req *pb.CreateBirthdayRequest) (*pb.BirthdayObject, error) {
-	birthdayCollection := mongoConnect.CNX.Database("birthdayDB").Collection("birthdays")
 
 	b := bson.M{
-		"name":           req.Name,
-		"date":           req.Date,
-		"personalNumber": req.PersonalNumber,
+		nameParameter:           req.Name,
+		dateParameter:           req.Date,
+		personalNumberParameter: req.PersonalNumber,
 	}
 
-	cursor, err := birthdayCollection.InsertOne(ctx, b)
+	cursor, err := s.BirthdayCollection.InsertOne(ctx, b)
 	if err != nil {
-		fmt.Println("error: ", err)
+		log.Fatal(err)
 	}
 
-	birthday := birthdayCollection.FindOne(ctx, bson.M{"_id": cursor.InsertedID})
-	BirthdayModel := &BirthdayStruct{}
+	birthday := s.BirthdayCollection.FindOne(ctx, bson.M{"_id": cursor.InsertedID})
+	BirthdayModel := &BirthdayModel{}
 	birthday.Decode(BirthdayModel)
 
 	convertedBirthday := BirthdayModel.asBirthdayObject()
-	fmt.Println(convertedBirthday)
 
 	return convertedBirthday, nil
 }
 
+// GetBirthday is getting a birthday object by personalNumber parameter
 func (s *Service) GetBirthday(ctx context.Context, req *pb.GetBirthdayRequest) (*pb.BirthdayObject, error) {
-	birthdayCollection := mongoConnect.CNX.Database("birthdayDB").Collection("birthdays")
 
-	birthday := birthdayCollection.FindOne(ctx, bson.M{"personalNumber": req.PersonalNumber})
-	BirthdayModel := &BirthdayStruct{}
+	birthday := s.BirthdayCollection.FindOne(ctx, bson.M{personalNumberParameter: req.PersonalNumber})
+	BirthdayModel := &BirthdayModel{}
 	birthday.Decode(BirthdayModel)
 	convertedBirthday := BirthdayModel.asBirthdayObject()
 
 	return convertedBirthday, nil
 }
 
+// GetAllBirthdays is creating all birthday objects
```

Comment:
```suggestion
// GetAllBirthdays returns all birthday objects in the DB
```
The documentation was wrong

Reply:
done